fvm/blueprints/fees.go

```diff
@@ -188,7 +189,7 @@ transaction(surgeFactor: UFix64, inclusionEffortCost: UFix64, executionEffortCos
 // SetExecutionEffortWeightsTransaction creates a transaction that sets up weights for the weighted Meter.
 func SetExecutionEffortWeightsTransaction(
 	service flow.Address,
-	weights map[uint]uint64,
+	weights weightedMeter.ExecutionWeights,
```

Comment:
You meant moving it to the `fvm/meter` next to the `Meter` interface right. That would touch a few more files in this PR. I am a bit mindful of that, since I want to port it to v0.25. Is it still ok with you.

Reply:
I actually went the other way around and removed this change, to keep the PR as small as possible.

I will open a new PR to clean this up.